.github/workflows/zeebe-daily-qa.yml

```diff
@@ -78,9 +78,11 @@ jobs:
             generation_template: ${{ format('Camunda {0}.0', needs.get-versions.outputs.third-last-version) }}
           - branch: ${{ format('stable/{0}', needs.get-versions.outputs.fourth-last-version) }}
             generation_template: ${{ format('Camunda {0}.0', needs.get-versions.outputs.fourth-last-version) }}
+    runs-on: ubuntu-latest
     name: Daily QA
-    uses: ./.github/workflows/zeebe-qa-testbench.yaml
-    with:
-      branch: ${{ matrix.branch }}
-      generation: ${{ matrix.generation_template }}
-    secrets: inherit
+    steps:
+      - uses: actions/checkout@v4
+      - name: Trigger QA on ${{ matrix.branch }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: gh workflow run --ref ${{ matrix.branch }} qa-testbench.yaml -F generation='${{ matrix.generation_template }}' -F branch=${{ matrix.branch }}
```

Comment:
```suggestion
        run: gh workflow run --ref ${{ matrix.branch }} zeebe-qa-testbench.yaml -F generation='${{ matrix.generation_template }}' -F branch=${{ matrix.branch }}
```

Reply:
oh we need to rename them on the stable branches then as well

Reply:
:see_no_evil:

Reply:
suggested to backport the rename instead here https://github.com/camunda/zeebe/pull/16559#issuecomment-1971140182

Reply:
ok one more run with all workflows renamed 🤞 
https://github.com/camunda/zeebe/actions/runs/8100106613